.jshintrc

```diff
@@ -9,6 +9,8 @@
   "strict": "global",
   "undef": true,
   "unused": true,
+  "esversion": 6,
```

Comment:
Is this wise? I know we're using some ES6 features in the tooling, but we don't currently rewrite ES6 in the application source code, do we?